config/locales/common/en.yml

```diff
@@ -63,6 +63,7 @@ en:
   or: Or
   other_info: Other Info
   oversize_feedback_file: 'Error: feedback file was %{file_size} larger than the maximum file size of %{max_file_size} MB'
+  oversize_submission_file: This file is too large to display. Download it instead.
```

Comment:
Move this string to under `views/submissions/en.yml`. There's a very similar `cannot_display` string. Note that the second sentence in that string should be replicated here: "Please download it to view."